src/snowcli/__about__.py

```diff
@@ -1,3 +1,3 @@
 from __future__ import annotations
 
-VERSION = "2.0.0-dev"
+VERSION = "2.0.0-alpha"
```

Comment:
Maybe we should think about `2.0.0-alpha.1`? WDYT?
How do we want version releases with fixes?

Reply:
@sfc-gh-turbaszek WDYT about this?